pkg/limatmpl/locator.go

```diff
@@ -0,0 +1,166 @@
+package limatmpl
```

Comment:
Maybe:
```suggestion
package templatelocator
```

Reply:
It may not make sense right now, but there will be a `limatmpl/assemble.go` in the future that contains all methods on a `limatmpl.Template` receiver, and then the package name `templatelocator` doesn't make sense anymore.

And while I haven't written it yet, the new `FillDefaults()` will also have to move to the `limatmpl` package (to avoid a circular dependency with `usrlocalsharelima`, which indirectly needs `limayaml`).

So for now at least I would like to keep the `limatmpl` package name.